elements/pl-file-preview/pl-file-preview.py

```diff
@@ -28,20 +29,13 @@ def render(element_html, data):
     # Pass through format errors from the file input elements
     html_params["errors"] = data["format_errors"].get("_files", [])
 
-    # Decode and reshape files into a useful form
+    # Reshape files into a useful form
     if len(submitted_files) > 0:
         files = []
```

Comment:
Files can now be assembled in one line (and possibly some of the logic above can be simplified?)
```suggestion
        files = [
            {
                    "name": file["name"],
                    "index": idx,
            }
            for idx, file in enumerate(submitted_files)
        ]
```